src/vm/peimagelayout.cpp

```diff
@@ -363,11 +399,65 @@ ConvertedImageLayout::ConvertedImageLayout(PEImageLayout* source)
 
 #ifdef CROSSGEN_COMPILE
     if (HasNativeHeader())
-        ApplyBaseRelocations();
+        ApplyBaseRelocations(false, false);
 #endif
 }
 
-MappedImageLayout::MappedImageLayout(HANDLE hFile, PEImage* pOwner)
+MinimalMappedImageLayout::MinimalMappedImageLayout(LPCWSTR assemblyPath)
```

Comment:
What is the rationale behind naming this layout "Minimal"? Can you please explain that?

Reply:
`MinimalMappedImageLayout` holds pointer to mapped assembly, which will be used later to initialize `MappedImageLayout`. As `MinimalMappedImageLayout` is only used for this, I thought it was good to name it as some kind of minimal version of `MappedImageLayout`. I think it could also be called `PreloadedMappedImageLayout` or something like that. What do you think about this?

Reply:
That would be a great name for that.